core/templates/base-components/oppia-footer.component.ts

```diff
@@ -16,22 +16,31 @@
  * @fileoverview Component for the footer.
  */
 
-import { Component } from '@angular/core';
+import { Component, ViewEncapsulation } from '@angular/core';
 import { Router } from '@angular/router';
 import { downgradeComponent } from '@angular/upgrade/static';
 import { PlatformFeatureService } from 'services/platform-feature.service';
 
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
 import { AppConstants } from 'app.constants';
+import { AlertsService } from 'services/alerts.service';
+import { ThanksForSubscribingModalComponent } from './thanks-for-subscribing-modal.component';
+import { MailingListBackendApiService } from 'domain/mailing-list/mailing-list-backend-api.service';
+
 import './oppia-footer.component.css';
 
 
 @Component({
   selector: 'oppia-footer',
   templateUrl: './oppia-footer.component.html',
-  styleUrls: ['./oppia-footer.component.css']
+  styleUrls: ['./oppia-footer.component.css'],
+  encapsulation: ViewEncapsulation.None,
```

Comment:
ViewEncapsulation set to None is not something we like to go for. This can lead to a whole host of issues if not done properly. For a component that uses ViewEncapsulation as None, the css styles need to be manually scoped. It is usually achieved by wrapping the entire component in a div with a class named after the component and using that class to scope the css. However, doing that on a top level component is too risky as we may end up affecting lower level non encapsulated components. Please try to move the problematic code (I assume links with translation were causing the problem) to a smaller re-usable component with no view encapsulation and the correct manually scoped.

Reply:
I have moved those translations to a separate reusable component. PTAL